Automattic/jetpack
# Instant Search: add more styling specificity to search input

Fixes https://github.com/Automattic/jetpack/issues/14608

#### Changes proposed in this Pull Request:
* Ensure there's a cross to clear search on browsers that support it.
* Reduce font-size on results count text.
* Make close button behave similarly on different themes.

![image](https://user-images.githubusercontent.com/390760/74433149-c5732380-4e57-11ea-9102-e64397d67d2e.png)


#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* No.

#### Testing instructions:

1. Follow [these setup instructions](https://github.com/Automattic/jetpack/blob/instant-search-master/modules/search/instant-search/README.md#testing-instructions) to set up Jetpack Instant Search on your site.
2. Search your site to trigger the Jetpack Search overlay.
3. Ensure that the Jetpack Search input field, the result count text, and the close button are styled correctly on multiple themes.

#### Proposed changelog entry for your changes:
* None.
